packages/playwright-core/types/types.d.ts

```diff
@@ -17482,6 +17482,11 @@ export interface APIRequest {
      */
     extraHTTPHeaders?: { [key: string]: string; };
 
+    /** 
+     * An object containing an option to throw an error when API request returns status codes other than 2xx and 3xx.
+     */
+    apiRequest?: {failOnStatusCode?: boolean;}
```

Comment:
This file is auto generated from `docs/`, don't edit it manually. Once you add references to `context-option-apirequest` in corresponding classes, this file will be automatically updated when you run build.

See `context-option-extrahttpheaders` for example.